microraiden/microraiden/webui/microraiden/package.json

```diff
@@ -17,7 +17,7 @@
     "web3"
   ],
   "contributors": [
-    "André Vitor de Lima Matos <andre@brainbot.com>"
+    "André Vitor de Lima Matos <andre@brainbot.li>"
```

Comment:
you have `.li` address too?

Reply:
Yes, I do, and as @loredanacirstea requested, I'll use it for documentation/contact/contribution